j2k/newSrc/org/jetbrains/kotlin/j2k/NewCodeBuilder.kt

```diff
@@ -249,6 +249,15 @@ class NewCodeBuilder {
 
         override fun visitMethodCallExpression(methodCallExpression: JKMethodCallExpression) {
             printer.printWithNoIndent(FqName(methodCallExpression.identifier.fqName).shortName().asString())
+            if (methodCallExpression.typeArguments.isNotEmpty()) {
```

Comment:
Use `renderList`, and take a look at how other type-arguments rendering was done, maybe it worth to extract  logic into function